lib/std/spinlock.zig

```diff
@@ -11,7 +12,8 @@ pub const SpinLock = struct {
         spinlock: *SpinLock,
 
         pub fn release(self: Held) void {
-            assert(@atomicRmw(u8, &self.spinlock.lock, builtin.AtomicRmwOp.Xchg, 0, AtomicOrder.SeqCst) == 1);
+            // TODO: @atomicStore() https://github.com/ziglang/zig/issues/2995
```

Comment:
FYI to anyone paying attention to this PR - this issue is contributor friendly if anyone wants to give it a try.